internal/provider/metric_export_prometheus_config_resource.go

```diff
@@ -0,0 +1,405 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/cockroachdb/cockroach-cloud-sdk-go/pkg/client"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"net/http"
+	"strings"
+)
+
+var metricExportPrometheusConfigAttributes = map[string]schema.Attribute{
+	"id": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: "Cluster ID.",
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplace(),
+		},
+	},
+	"enabled": schema.BoolAttribute{
+		Default:     booldefault.StaticBool(true),
+		Optional:    true,
+		Computed:    true,
+		Description: "Represents prometheus integration status. Please refrain setting any value as it is used for state management.",
+	},
+	"status": schema.StringAttribute{
+		Computed:    true,
+		Description: "Encodes the possible states that a metric export configuration can be in as it is created, deployed, and disabled.",
```

Comment:
We may want to list out the actual states here.   How about this wording instead:

```
Description: "The current state of the metric export configuration.  Possible values are: " + strings.Join(client.AllowedMetricExportStatusTypeEnumValues, ", "),
```

Reply:
This is addressed.

Reply:
Nice, thank you!